### PR TITLE
Fix PickerWindows (and DatePickerExample Page)

### LINKED ISSUE #4597: (0.62) Exception thrown any time pickers are imported
We see an invariant violation on pages with pickers due to RCTPicker being registered twice. We've seen a similar issue during the 0.62 upgrade for ScrollView that required taking in some new JS-side changes for code that only happens off of Android/iOS.

### PR DESCRIPTION
**Pull request by NickGerleman**
Fixes #4597

Our implementation of Picker is a bit deformed. In Stock React Native, Picker.js will delegate to PickerAndroid and PickerIOS. In Windows, we the two are copy pasted. In 0.62, invariants are added to prevent duplicate view manager registration. We run into issues with this with Picker, since our copies will both call requireNativeComponent. This change modifies the normal picker to delegate to PickerWindows to work correctly from the stock picker. I.e. The stock picker's child views are meant for data only and will throw if rendered, where the Windows version's are included but used to noop render.

Picker is getting lean-cored, so I didn't invest much into fixing PickerWindow's isssues, converting to Flow, NativeComponent strucutre, etc.

Validated Picker, PickerWindows, DatePicker RNTester pages work correctly.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/4613)

### CODE REVIEW

File: vnext/src/Libraries/Components/Picker/PickerWindows.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const RCTPicker = requireNativeComponent<
 
 class PickerItem extends React.Component<IPickerItemProps> {
   public render(): JSX.Element | null {
-    return null;
+    throw null;
```
Comment (NickGerleman) [inline vnext/src/Libraries/Components/Picker/PickerWindows.tsx:22]: `throw null` is kind of crazy, but matches upstream behavior/behavior in Picker.js

File: vnext/src/Libraries/Components/Picker/PickerWindows.tsx [hunk 2]
```diff
@@ -85,20 +85,19 @@ export class Picker extends React.Component<IPickerProps, State> {
   }
 
   public render(): JSX.Element {
-    const props = {...this.props};
-    props.onStartShouldSetResponder = () => true;
-    props.onResponderTerminationRequest = () => false;
-    props.style = this.props.style;
```
Comment (NickGerleman) [inline vnext/src/Libraries/Components/Picker/PickerWindows.tsx:91]: This line doesn't seem to do anything useful. We already do a shallow clone of props including `props.style` with `const props = {...this.props};` above.

File: vnext/src/Libraries/Components/Picker/PickerWindows.tsx [hunk 3]
```diff
@@ -85,20 +85,19 @@ export class Picker extends React.Component<IPickerProps, State> {
...
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (NickGerleman) [inline vnext/src/Libraries/Components/Picker/PickerWindows.tsx:88]: There's a setting to disable this check when adjacent to the rest parameter that's off by default, but not in stock RN eslint rules that we're using apparently. I'd be fine changing that as well.

File: vnext/src/Libraries/Components/Picker/Picker.windows.js [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+      return <PickerIOS {...this.props}>{this.props.children}</PickerIOS>;
+    } else if (Platform.OS === 'android') {
+      return (
+        /* $FlowFixMe(>=0.81.0 site=react_native_android_fb) This suppression
+         * was added when renaming suppression sites. */
+        <PickerAndroid {...this.props}>{this.props.children}</PickerAndroid>
+      );
+    } else if (Platform.OS === 'windows') {
+      return (
+        <PickerWindows {...this.props}>{this.props.children}</PickerWindows> // [Windows]
```
Comment (vmoroz) [inline vnext/src/Libraries/Components/Picker/Picker.windows.js:152]: >// [Windows] [](start = 77, length = 12)

Could we remove it? It seems to bring to value.

Comment (NickGerleman) [reply vnext/src/Libraries/Components/Picker/Picker.windows.js:152]: We've been using that to annotate differences in Windows patches. E.g. to ctrl+F for without opening a diff tool.

**Pull request merged**